crates/bevy_asset/src/handle.rs

```diff
@@ -230,6 +231,34 @@ impl<T: Asset> Clone for Handle<T> {
     }
 }
 
+impl<T: Asset + 'static> Lerp for Handle<T> {
```

Comment:
I'm a bit nervous about chucking Lerp and related methods onto *all* handles to assets. What if we end up using the asset system for audio? Or dialog?

Perhaps an extra trait + trait bound makes sense here, to limit this to assets with a transform.

Reply:
Handle fields can only be swapped, because `lerp` can't create a new asset handle to return, that's by desing.

What you are describing is done via `impl<T> Lerp for T` not by `impl<T> Lerp for Handle<T>`, even still I think lerping should only be done in copy types;

A new PR could add a trait that describes `fn lerp(&a, &b, t, &mut target)`